corehq/apps/users/models.py

```diff
@@ -1155,13 +1155,14 @@ def wrap_correctly(cls, source):
             }[source['doc_type']].wrap(source)
 
     @classmethod
-    def get_by_username(cls, username):
+    @skippable_quickcache(['username'], skip_arg='strict')
```

Comment:
Very subjective, but I feel like I generally prefer to see timeout values explicit rather than have to lookup/know what the default is in quickcache